object_tracking_with_gcp/requirements.txt

```diff
@@ -0,0 +1,4 @@
+google-cloud-storage
```

Comment:
Let's put it in ipynb as cell. And delete this.

Reply:
I'll just have `!pip install -r requirements.txt` in the jupyter-notebook. Seems like a cleaner solution.

Reply:
We want to put it into google colab also